pt/git.md

```diff
@@ -37,9 +37,12 @@ Maus exemplos seriam: `maria_22_de_abril`, `regras`.
 
 ## [Mensagens de *commit*](https://xkcd.com/1296/)
 
+Use [estas diretrizes](https://cbea.ms/git-commit/) (em inglês).
+
 Dentro do possível, mantenha seus *commits* bem descritos:
-- comece todos as suas mensagens de *commit* com a tag `[pt]`;
-- prefira escrevê-las em inglês;
-- a primeira linha da sua mensagem não deve ultrapassar 50 caracteres.
+- comece todas as suas mensagens de *commit* com a tag `[pt]`;
+- escreva-as em inglês;
+- a primeira linha da sua mensagem não deve ultrapassar 50 caracteres;
+- a não ser que suas mudanças sejam gerais (ou seja, se apliquem a várias regras), busque pôr o nome da regra na primeira linha da mensagem.
 
-O adágio "*commit early, commit often*" só se aplica ao seu trabalho **local**.
+O adágio "*commit early, commit often*" só se aplica ao seu trabalho **local**. Ao criar um *pull request*, o ideal é que ele seja composto de poucos *commits* razoavelmente completos. Prefira fazer *squash* se você tiver muitos *commits* com mensagens do tipo `"[pt] Fix error"`, `"[pt] Add antipattern"`.
```

Comment:
See [this review](https://github.com/languagetool-org/languagetool-org.github.io/pull/17/files#diff-136793defa5a8800be239403d67914819ef28274e337282001b8c20c355f30b0)